mobile/src/main/java/rs/readahead/washington/mobile/domain/repository/ITellaUploadsRepository.java

```diff
@@ -0,0 +1,22 @@
+package rs.readahead.washington.mobile.domain.repository;
+
+import java.util.List;
+
+import io.reactivex.Completable;
+import io.reactivex.Single;
+import rs.readahead.washington.mobile.domain.entity.MediaFile;
+
+public interface ITellaUploadsRepository {
+    Completable scheduleUploadMediaFiles(List<MediaFile> mediafiles);
+    Single<List<MediaFile>> getUploadMediaFiles(UploadStatus status);
+    Completable setUploadStatus(long mediaFileId, UploadStatus status);
+    Completable setUploadReschedule(long mediaFileId);
+    Completable setUploadedAmount(long mediaFileId, long uploadedAmount);
```

Comment:
IMHO name of this method (and implementation support methods) is bad and should be changed - we are not setting any 'amount' here but updating uploading info: uploaded size (and status also, as per other comment that should be one method updating uploading info and setting updated time for that).